angular/src/app/discovery/beacon/beacon-search.component.scss

```diff
@@ -80,6 +80,12 @@ ddap-beacon-search-result-table {
       width: 100%;
     }
   }
+
+  mat-drawer {
+    width: 100vw;
+    max-width: 100vw;
+    min-width: 0;
```

Comment:
Giving that the width is fixed, I don't think we need this.

Reply:
I needed to override the normal css (no media query)
```
mat-drawer {
  min-width: 30rem;
  max-width: 30rem;
}
```
But I guess there's no reason for it to be like that. Might as well change it to `width: 30rem`